app/src/main/java/com/jnj/vaccinetracker/common/data/models/Constants.kt

```diff
@@ -80,7 +80,7 @@ object Constants {
 
     const val CONCEPT_NAME_WEIGHT_FOR_AGE_Z_SCORE = "Weight for age Z score"
     const val CONCEPT_NAME_HEIGHT_FOR_AGE_Z_SCORE = "Height for age Z score Vxnaid"
-    const val CONCEPT_NAME_MUACA_Z_SCORE = "MUACA Vxnaid"
```

Comment:
this is the name of the concept in the system. Please do not change it, because system will not find the concept with "MUAC Vxnaid"

Reply:
you can change only the name of the constant but please do not change the value

Reply:
Opps i went ahead and edited it among concepts.